packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -71,7 +71,8 @@ const generateImagePlaceholder = (alt, isAriaHidden, variant) => {
  */
 const generateImage = (image, alt, isAriaHidden, variant, handleFallback) => {
   const icon = generateImagePlaceholder(alt, isAriaHidden, variant);
-  return <TerraImage className={cx('image')} src={image} placeholder={icon} alt={alt} onError={handleFallback} />;
+  const imageFit = (variant === 'user') ? ImageFitTypes.COVER : undefined;
```

Comment:
The facility variant accepts an image prop too. Also, lets use the defined variants since they exist:

```suggestion
  const imageFit = (variant === AVATAR_VARIANTS.user || variant === AVATAR_VARIANTS.facility) ? ImageFitTypes.COVER : undefined;
```

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/700f775f5918c4a5babb6323f168d75b49205ec8.

Reply:
@bjankord Should we add test cases for non-square image for facility variant as well?